db/migrate/20220501021745_add_website_url_and_contact_role_to_businesses.rb

```diff
@@ -0,0 +1,6 @@
+class AddWebsiteUrlAndContactRoleToBusinesses < ActiveRecord::Migration[7.0]
+  def change
+    add_column :businesses, :website_url, :string, null: true
+    add_column :businesses, :contact_role, :string, null: true
```

Comment:
```suggestion
    add_column :businesses, :website, :string, null: true
    add_column :businesses, :role, :string, null: true
```

And applied to the rest of the PR, too!

Reply:
I was going to name the field just `website` but then I thought it was better to be more specific in case you wanted to add a `website_name` or `website_icon` in the future.

The column name `role` seemed to vague to me, If I see `business.role` I would expect it to return the role of the business. Naming it `contact_role` seems like a much clearer name in my opinion.

Reply:
> I was going to name the field just `website` but then I thought it was better to be more specific in case you wanted to add a `website_name` or `website_icon` in the future.

If/when we add website name or icon we can rename this column.

> The column name `role` seemed to vague to me, If I see `business.role` I would expect it to return the role of the business. Naming it `contact_role` seems like a much clearer name in my opinion.

I buy that! Then we should probably rename `name` to `contact_name`.

Reply:
I have pushed a commit to rename business `name` to `contact_name`

https://github.com/joemasilotti/railsdevs.com/pull/397/commits/50e5ef99258db620fb71a1a41ebe3a10a39feb5a

Reply:
I'd prefer to rename this in a separate PR. But if we are going to do it here then let's not alias the method name and write a migration to rename the column.

Reply:
I updated the migration to rename the database field.

https://github.com/joemasilotti/railsdevs.com/pull/397/files#diff-5b5a85582ac4960aa6ae66f16491a3fb6878af9fea738f5dcd6e65c89bdf5182R5